portal/templates/redesign/help-and-support_new.html

```diff
@@ -34,29 +37,224 @@ <h1 class="banner__text--primary">Help and support</h1>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q13" type="checkbox" />
+    <label for="FAQ-q13">
+            <strong>I've set up my students. How do they log in?</strong>
+    </label>
+    <answer>
+            <p>School students log in via the <a href=\"{% url 'play' %}\">Play</a> page, using their name, class access code, and password to sign in.</p>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q14" type="checkbox" />
+    <label for="FAQ-q14">
+            <strong>As a student in a school or club, who can I share my custom created levels with?</strong>
+     </label>
+        <answer>
+            <p>You can only share levels with other students in your class. Any level you create will be automatically shared with your teacher.</p>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q15" type="checkbox" />
+    <label for="FAQ-q15">
+            <strong>As a teacher, who can I share my custom created levels with?</strong>
+     </label>
+        <answer>
+            <p>As a teacher, you will be able to share levels you create with your classes and with other teachers in the school.
+            <br>If another teacher shares a level with you that you'd like to share with one of your classes, you'll need to save it as a new level which is owned by you first by loading and saving it in 'Create'.</p>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q16" type="checkbox" />
+    <label for="FAQ-q16">
+            <strong>How can I control the levels my students create?</strong>
+    </label>
+        <answer>
+            <p>Students can only share levels within their class and as a teacher you will see a copy of the level automatically.
+            <br>You have the ability to play or delete these levels.To share such a level to other teachers or students, you'll need to save it as a new level which is owned by you first by loading and saving it in 'Create'.</p>
+        </answer>
+</div>
+
+<div class="body-text dropdown">
+    <input id="FAQ-q17" type="checkbox" />
+    <label for="FAQ-q17">
+            <strong> I am an independent student. Why can't I share my custom created levels?</strong>
+     </label>
+        <answer>
+            <p>Students can only share levels within their class and as a teacher you will see a copy of the level automatically.  You have the ability to play or delete these levels.
+            <br>To share such a level to other teachers or students, you'll need to save it as a new level which is owned by you first by loading and saving it in 'Create'.</p>
+        </answer>
+</div>
```

Comment:
This section needs to be indented by two tabs, just to respect the indentation in the file :)